app/Http/Controllers/PortalsController.php

```diff
@@ -94,6 +94,7 @@ public function importUsers(Request $request): JsonResponse
             $person->firstname = $user['firstname'];
             $person->lastname = $user['lastname'];
             $person->email = $user['email'];
+            $person->pin = $user['matrikelNr'];
```

Comment:
```suggestion
            $person->pin = $user['pin'];
```

I think we should name the portals parameter also "pin". Because we should not save the whole matrikelNr.